docs/End_to_End_Testing.md

```diff
@@ -0,0 +1,49 @@
+#End-to-End Testing
+
+End-to-end test must be included as a part of a PR for any contribution to Flux including: [Flux Functions](https://github.com/Anaisdg/flux/blob/contributing/docs/Flux_Functions.md), [Standalone Scalar Functions](https://github.com/Anaisdg/flux/blob/contributing/docs/Scalar_Functions.md), [Sink/Source Functions](https://github.com/Anaisdg/flux/blob/contributing/docs/Source_Sink_Functions.md), or [Stream Transformation Functions](https://github.com/Anaisdg/flux/blob/contributing/docs/Stream_Transformation_Functions.md). 
+
+## Required guidelines
+
+Please help us make the contribution process easier by providing feedback about your experience and any technical hurdles you encountered here.
+
+### **Pure Flux Code Functions Guidelines**
+
+If you add a new ***.flux file to stdlib/testing/testdata, our test framework will automatically find it and run the test, if you've written it correctly. Consider the anatomy of simple_max.flux:

```

Comment:
When you add a new .flux file (with correct form and syntax) to stdlib/...., our test framework automatically detects the file and tests it. To ensure your file passes the test, consider the correct form of simple_max.flux: